forc-plugins/forc-client/src/cmd/call.rs

```diff
@@ -223,6 +223,10 @@ pub struct Command {
     /// The output format to use; possible values: default, raw
     #[clap(long, default_value = "default")]
     pub output: OutputFormat,
+
+    /// Output call receipts
+    #[clap(long, alias = "receipts")]
```

Comment:
Maybe we should also have a short for this one, seems like a very handy feature.

Reply:
Something like `-r` maybe?

Reply:
https://github.com/FuelLabs/sway/pull/6968/commits/f3972ae3c0f278b515151f3c06f9b51cff61a865